Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -371,14 +371,17 @@ namespace gdjs {
       }
       const previousAction = this._action;
       this._action = this._animationMixer.clipAction(clip);
+
+      //We stop action and set enabled to true to fix troubles in animation switching and looping coming from this._action not being reset.
```

Comment:
Could you extend this comment to explain a bit more why this is needed? It feels like there's a strong reason why we're doing this, but by reading the comment, I don't quite get it.
You can write on multiple lines to really explain what is happening and why we're doing this